ibm/resource_ibm_bandwidth_pool.go

```diff
@@ -50,16 +39,20 @@ func resourceIBMBandwidthPoolCreate(d *schema.ResourceData, meta interface{}) er
 	sess := meta.(ClientSession).SoftLayerSession()
 	log.Println("ordering bandwidth Pool service...")
 	///get the value of all the parameters
-	accountID := d.Get("accountId").(int)
-	bandwidthAllotmentTypeID := d.Get("bandwidthAllotmentTypeId").(int)
 	name := d.Get("name").(string)
 	locationGroupID := d.Get("locationGroupId").(int)
 	///creat an object of Bandwidth Pool service
 	service := services.GetNetworkBandwidthVersion1AllotmentService(sess)
-	//////pass the parameters to create bandwidth pool
+	account := services.GetAccountService(sess)
+	// get the account Id
+	accountData, err := account.Mask("id").GetObject()
+	if err != nil {
+		return fmt.Errorf("%s", err)
```

Comment:
Can we have err with proper message?
eg. fmt.Errorf("Encountered problem trying to get the account ID: %s", err)